packages/react-ds/src/hoc/WithField.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { HTMLProps, forwardRef, FunctionComponent } from 'react'
+
+import classNames from 'classnames/bind'
+import uniqid from 'uniqid'
+
+import { IField } from '@venice/core/models/field.model'
+import styles from '@venice/styles/components/Field.module.scss'
+
+import Alert from '../components/Icons/Alert'
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const WithField = (Component: FunctionComponent<any>) =>
```

Comment:
It works?

```ts
interface IWithFieldProps<T> {
   value: T
}

const MyComponent: FunctionComponent<IWithFieldProps<string>> = ({ value }) => <div />;
```

Reply:
No
"/

Reply:
`const WithField = <P extends Object>(Component: FunctionComponent<P>)` not too?

Reply:
neither
<img width="968" alt="Screen Shot 2020-06-25 at 13 35 43" src="https://user-images.githubusercontent.com/2536827/85761277-f9edca80-b6e8-11ea-8bc5-66e8ed302ea3.png">
<img width="1132" alt="Screen Shot 2020-06-25 at 13 37 42" src="https://user-images.githubusercontent.com/2536827/85761353-0d993100-b6e9-11ea-9cf3-d6267a1d7fec.png">
<img width="1127" alt="Screen Shot 2020-06-25 at 13 36 28" src="https://user-images.githubusercontent.com/2536827/85761254-f5291680-b6e8-11ea-8bda-29d35a005c2d.png">